initramfs-init.in

```diff
@@ -281,6 +267,32 @@ is_url() {
 	esac
 }
 
+/bin/busybox mkdir -p /usr/bin /usr/sbin /proc /sys /dev $sysroot \
+        /media/cdrom /media/usb /tmp /run
+
+# Spread out busybox symlinks and make them available without full path
+/bin/busybox --install -s
+export PATH=/usr/bin:/bin:/usr/sbin:/sbin
+
+# If we dont pre-create /dev/null, the command mounting the devtmpfs will create it
+# implicitly as an file with the "2>" redirection
+mknod -m 666 /dev/null c 1 3
```

Comment:
the `/dev/null` may exist in the initramfs image and if it does it will print an error:
````
mknod: /dev/null: File exists
````
So I think we need to test if it exists with
````sh
[ -c /dev/null ] || mknod -m 666 /dev/null c 1 3
````

Reply:
I re-added the check, plus documentation for it.